src/StrongName.targets

```diff
@@ -0,0 +1,49 @@
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. -->
+<Project>
+
+  <!-- See https://github.com/dotnet/arcade/blob/dbb4142ebc34d0a4b9f26136f86065c037dc4b80/src/Microsoft.DotNet.Arcade.Sdk/tools/StrongName.targets#L33-L37 for where this was copied from -->
```

Comment:
here is the attribution from where we referenced

Reply:
can you add that for the props file?